src/clients/client.ts

```diff
@@ -177,61 +179,57 @@ class Client {
    * @method getTasks
    * @returns {Promise<Result<Task[]>>} - Promise returning all tasks
    */
-  async getTasks(): Promise<Result<Task[]>> {
-    return await this.tasks.getClientTasks()
+  async getTasks(params?: TaskParams): Promise<Result<Task[]>> {
+    return await this.tasks.getTasks(params)
   }
 
   /**
    * Get one task on the client scope
    * @memberof MeiliSearch
    * @method getTask
-   * @param {number} taskId - Task identifier
+   * @param {number} taskUid - Task identifier
    * @returns {Promise<Task>} - Promise returning a task
    */
-  async getTask(taskId: number): Promise<Task> {
-    return await this.tasks.getClientTask(taskId)
+  async getTask(taskUid: number): Promise<Task> {
+    return await this.tasks.getTask(taskUid)
   }
 
   /**
-   * Wait for a batch of tasks to be processed.
+   * Wait for multiple tasks to be finished.
+   *
    * @memberof MeiliSearch
    * @method waitForTasks
-   * @param {number[]} taskIds - Tasks identifier
+   * @param {number[]} taskUids - Tasks identifier
    * @param {WaitOptions} waitOptions - Options on timeout and interval
    *
    * @returns {Promise<Result<Task[]>>} - Promise returning an array of tasks
    */
   async waitForTasks(
```

Comment:
Do you use this method a lot? In the package?

Reply:
I would like to use it in Strapi so I thought it would be a cute addition. Do you prefer that I remove it?